frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -190,6 +150,10 @@ const TranslationPage = () => {
           </div>
         </div>
       </div>
+      <Autosave
```

Comment:
out of curiosity, how do you think we can scale autosaving if user testing finds that comments should be auto save on edit?

Reply:
As in per keystroke? Wouldn't we just revert back to the original state?

Reply:
Ideally we would avoid per keystroke saving again. Would we be able to use another `<Autosave>` for comment changes?

Reply:
Ohh for comments. I'm thinking the easiest way would be that `changedStoryLines` becomes something like:
```json
changesToSave = {
  "storyLines": StoryLine [],
  "comments": Comment [],
}
```
and is still passed as a prop. Call executes none, one, or both mutations as necessary? Alternatively, we could introduce another state variable exclusively for comments and pass both into the Autosave component as props. The latter would probably be better for legibility, but I'm not sure whether it's worth introducing another state variable when changesToSave does make sense as a group + we want to execute mutations together in calls.

Reply:
This sounds good to me - the only downside in UX is if the user edits text and then immediately edits a comment, the text save would be delayed until the comment changes stop long enough to be saved.
We currently have two different mutations but I think that's fine - the case of both story translation content and comments being edited within a 1000ms period is probably pretty rare

Reply:
Agree it's an edge case still worth looking at. The mutations should be together on the frontend I think? but will need distinct resolving + database locks in backend

```
mutation {
  mutation1() {}
  mutation2() {}
}
```